docs/source/python/extending_types.rst

```diff
@@ -225,22 +280,22 @@ of the given frequency since 1970.
             # attributes need to be set first before calling
             # super init (as that calls serialize)
             self._freq = freq
-            super().__init__(pa.int64(), 'my_package.period')
+            super().__init__(pa.int64(), "my_package.period")
 
         @property
         def freq(self):
             return self._freq
 
         def __arrow_ext_serialize__(self):
-            return "freq={}".format(self.freq).encode()
+            return "freq={self.freq}".encode()
```

Comment:
Please revert the change to this one line. It was correct before but now it is not.